webui/react/src/components/Avatar.tsx

```diff
@@ -1,45 +1,18 @@
-import { Tooltip } from 'antd';
 import React, { useEffect, useState } from 'react';
 
 import { useStore } from 'contexts/Store';
 import { useFetchUsers } from 'hooks/useFetch';
-import { hex2hsl, hsl2str } from 'shared/utils/color';
-import md5 from 'shared/utils/md5';
+import Avatar, { Props } from 'shared/components/Avatar/Avatar';
 import { getDisplayName } from 'utils/user';
 
-import css from './Avatar.module.scss';
-
-interface Props {
-  hideTooltip?: boolean;
-  large?: boolean;
+const UserAvatar: React.FC<{
   name?: string;
   // TODO: separate components for
```

Comment:
I didn't get the second part of the todo that's why I didn't touch it. I'm not sure if we're resolving that here.

Reply:
oh i think i just meant that it should be split up between 1) Avatar and AvatarCard that just displays based on string and 2) parent components that also include looking up the user's data. i mentioned the profile image but that was just in case it got added to Avatar before this split happened. so this TODO should be addressed here.

Reply:
one other thing i noticed here is that i think the `name` prop is only really used in the storybook story. i was going to remove it and update the story but never got around to it. definitely doesn't have to be included in this PR but just mentioning in case you feel like addressing.

Reply:
sure thing. my solution: https://github.com/determined-ai/determined/pull/4430/commits/a9212ecd76a4fd54befd673ea64bd2070c3d8deb